lib/codeowners/checker.rb

```diff
@@ -110,22 +152,35 @@ def consistent?
 
     def commit_changes!
       @git.add(codeowners_filename)
+      @git.add(ownerslist_filename)
       @git.commit('Fix pattern :robot:')
     end
 
+    def ownerslist_filename
```

Comment:
I'm not sure about this method name:

```suggestion
    def teams_filename
```

Reply:
Now I see that it also includes people. Maybe we could call it `owners` since the other is the relationship with the code and this is only the cache of the owners' names.